tests/functional/services/transfer/endpoint_manager/test_endpoint_manager_acl_list.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+
+@pytest.mark.xfail
+def test_endpoint_manager_acl_list():
```

Comment:
Just to confirm:

* `endpoint_manager_` should be included in the filename and test name (here, and in other files in this directory)? Or should this be `test_acl_list()` or similar?
* The lack of an active verb in this particular name (and a few others) is anticipated, i.e. there's no "get" verb that should be here?

Reply:
These are intentional, but the test function name is something we didn't specify.

The pattern we agreed upon (sorry that you missed the relevant meeting!) was that we'd have a module named by the _method name_ on the relevant client.
So the method is defined as `TransferClient.endpoint_manager_acl_list`, which determines the module name without any `get_` prefix as `test_endpoint_manager_acl_list`.

Absent any decision about what to name the functions inside, I decided to keep them matching the method names. I don't think it matters _too much_ since these are mostly here to flag the missing-ness of any tests in the modules.